docs/encryption/README.md

```diff
@@ -0,0 +1,152 @@
+# Setting Up Encryption in K8ssandra
+
+Cassandra offers the ability to encrypt internode communications and client to node communications separately.
+We will explain here how to set up and configure encryption in k8ssandra clusters.
+
+## Generating Encryption Stores
+
+If you do not have a set of encryption stores available, follow the indications from [this TLP blog post](https://thelastpickle.com/blog/2021/06/15/cassandra-certificate-management-part_1-how-to-rotate-keys.html). More specifically, use [the script](https://github.com/thelastpickle/cassandra-toolbox/tree/main/generate_cluster_ssl_stores) they created to generate the SSL stores.
+
+A simplified procedure would be to clone the [cassandra-toolbox](https://github.com/thelastpickle/cassandra-toolbox) GitHub repository and create a `cert.conf` file with the following format:
```

Comment:
This is quite useful... I wonder what other goodness is in there that we should let folks know about? That's not really a question for now, just a thanks for the heads up, nice!